core/templates/components/concept-card/concept-card.template.html

```diff
@@ -1,6 +1,5 @@
 <div class="oppia-concept-card-viewer" ng-if="!$ctrl.loadingMessage">
   <div class="concept-card-explanation">
-    <b> Explanation </b>
```

Comment:
Any reason why we have removed this?

Reply:
I asked for this -- it was confusing to have the heading on the top when I was previewing it as a user!

Reply:
Ah okay got it !